packages/strapi-design-system/src/ToggleCheckbox/ToggleCheckbox.js

```diff
@@ -76,28 +66,39 @@ export const ToggleCheckbox = React.forwardRef(
         <VisuallyHidden>{children}</VisuallyHidden>
 
         <ToggleCheckboxWrapper background="neutral0" hasRadius size={size} disabled={disabled}>
-          <OffBox
-            background={disabled ? 'neutral150' : offCheckboxBackgroundColor}
+          <ValueBox
             paddingLeft={7}
             paddingRight={7}
             aria-hidden={true}
+            checked={checked === null ? false : !checked}
```

Comment:
What makes you think that we don't? I've expanded and added two stories about that - it will work in the future the same way it does today :)

https://design-system-git-content-73-toggle-checkbox-refresh-strapijs.vercel.app/?path=/story/design-system-components-toggleinput--null-value

Reply:
Alright, I thought that this condition  `checked === null ? false : !checked` would remove the null state